modules/spring-config/src/main/resources/META-INF/mule-module.properties

```diff
@@ -0,0 +1,32 @@
+module.name=spring-config
+
+artifact.export.classPackages=org.mule.runtime.config.spring,\
```

Comment:
The idea is to expose packages explicitly to make it clear what is exposed on the API